src/commands/util/event.js

```diff
@@ -0,0 +1,56 @@
+const Command = require('../Command');
+const fetch = require('node-fetch');
+
+const info = ':information_source:';
+const calendar = ':calendar_spiral:';
+const pushPin = ':round_pushpin';
+const clock = ':clock3:';
+
+module.exports = new Command({
+  name: 'event',
+  description: 'get number of events from a calendar',
+  category: 'information',
+  aliases: [],
+  permissions: 'general',
+
+  execute: async (message, args) => {
+    const numOfEvents = args[0];
+    const events = await addEvents(numOfEvents);
+    const embed = {
+      'title': 'Upcoming SCE Events!',
+      'url': 'http://sce.sjsu.edu/',
+      'color': 13110653,
+      'fields': events,
+    };
+    message.channel.send({ embed });
+  },
+});
+
+function addEvents(numOfEvents) {
+  return new Promise((resolve, reject) => {
+    let eventData = [];
+    let fields = [];
+    // eslint-disable-next-line
+    fetch(`http://localhost:8082/api/mailer/getCalendarEvents?calendarID=primary&numOfEvents=${numOfEvents}`)
```

Comment:
lets not hardcode the url of the google api on this line. instead we can:
1. define `SCE_GOOGLE_API_URL` as a field in [``](https://github.com/SCE-Development/SCE-discord-bot/blob/master/config.example.json) like:
```json
"SCE_GOOGLE_API_URL": "http://localhost:8082"
```

2. import it at the top of the file like:
```js
const { SCE_GOOGLE_API_URL } = require('../../config.json');
```
3. use it on this line like:
```suggestion
    fetch(`{SCE_GOOGLE_API_URL}/api/mailer/getCalendarEvents?calendarID=primary&numOfEvents=${numOfEvents}`)
```